### PR TITLE
[ioctl/newcmd] delete NewKeyStore parameteres in ioctl.Client

### PR DESCRIPTION
**Pull request by huof6829**
callers use the same default wallet dir and StandardScryptN  StandardScryptP.

### CODE REVIEW

File: ioctl/newcmd/account/account.go [hunk 1]
```diff
@@ -355,7 +353,7 @@ func storeKey(client ioctl.Client, privateKey, walletDir, password string) (stri
 
 	switch sk := priKey.EcdsaPrivateKey().(type) {
 	case *ecdsa.PrivateKey:
-		ks := client.NewKeyStore(walletDir, keystore.StandardScryptN, keystore.StandardScryptP)
+		ks := client.NewKeyStore()
```
Comment (CoderZhi) [inline ioctl/newcmd/account/account.go:356]: walletDir is not in use

Comment (huof6829) [reply ioctl/newcmd/account/account.go:356]: yes, useless parameter

**Comment by huof6829** [comment]
move keystore function to ioctl.Client

File: ioctl/client.go [hunk 2]
```diff
@@ -185,8 +188,31 @@ func (c *client) Address(in string) (string, error) {
...
+		defer func(k *ecdsa.PrivateKey) {
+			b := k.D.Bits()
+			for i := range b {
+				b[i] = 0
+			}
+		}(key.PrivateKey)
+	}
+	if err != nil {
+		return nil, output.NewError(output.KeystoreError, "failed to decrypt key", err)
+	}
```
Comment (CoderZhi) [inline ioctl/client.go:214]: move to line 203

**Review by CoderZhi — APPROVED** [review]

**Review by davidofell5 — APPROVED** [review]

**Review by davidofell5 — APPROVED** [review]

File: ioctl/client.go (hunk 2 above)
Comment (huof6829) [reply ioctl/client.go]: yes, check err first

**Review by Liuhaai — APPROVED** [review]

**Review by Scheusal13 — COMMENTED** [review]
0xEa5C920973eBfff95270dd52c8045c4e6eBFc847

**Pull request merged**